### ISSUE TITLE
test: raftstore::test_conf_change::test_server_split_brain failed

### ISSUE THREAD

**Issue by kennytm** [tikv/tikv#4356]
## Bug Report

**What version of Rust are you using?**

nightly-2018-12-06 (`rustc 1.32.0-nightly (14997d56a 2018-12-05)`)

**What operating system and CPU are you using?**

(The failure comes from the CI)

**What did you do?**

Wait for the CI report from #4348

**What did you expect to see?**

CI pass

**What did you see instead?**

[CI failed](https://internal.pingcap.net/idc-jenkins/blue/organizations/jenkins/tikv_ghpr_test/detail/tikv_ghpr_test/2248/pipeline/) in the unrelated test `raftstore::test_conf_change::test_server_split_brain`:

```
test raftstore::test_conf_change::test_server_split_brain ... thread 'raftstore::test_conf_change::test_server_split_brain' panicked at 'can't get value Some("v1") for key "k1"', components/test_raftstore/src/util.rs:64:5
note: Some details are omitted, run with `RUST_BACKTRACE=full` for a verbose backtrace.
stack backtrace:
   0: std::sys::unix::backtrace::tracing::imp::unwind_backtrace
             at src/libstd/sys/unix/backtrace/tracing/gcc_s.rs:49
   1: std::sys_common::backtrace::_print
             at src/libstd/sys_common/backtrace.rs:71
   2: std::panicking::default_hook::{{closure}}
             at src/libstd/sys_common/backtrace.rs:59
             at src/libstd/panicking.rs:211
   3: std::panicking::default_hook
             at src/libstd/panicking.rs:227
   4: std::panicking::rust_panic_with_hook
             at src/libstd/panicking.rs:491
   5: std::panicking::continue_panic_fmt
             at src/libstd/panicking.rs:398
   6: std::panicking::begin_panic_fmt
             at src/libstd/panicking.rs:353
   7: test_raftstore::util::must_get
             at components/test_raftstore/src/util.rs:64
   8: test_raftstore::util::must_get_equal
             at components/test_raftstore/src/util.rs:72
   9: integrations::raftstore::test_conf_change::test_split_brain
             at tests/integrations/raftstore/test_conf_change.rs:456
  10: integrations::raftstore::test_conf_change::test_server_split_brain
             at tests/integrations/raftstore/test_conf_change.rs:508
  11: integrations::raftstore::test_conf_change::test_server_split_brain::{{closure}}
             at tests/integrations/raftstore/test_conf_change.rs:505
  12: core::ops::function::FnOnce::call_once
             at /rustc/14997d56a550f4aa99fe737593cd2758227afc56/src/libcore/ops/function.rs:238
  13: <F as alloc::boxed::FnBox<A>>::call_box
             at src/libtest/lib.rs:1471
             at /rustc/14997d56a550f4aa99fe737593cd2758227afc56/src/libcore/ops/function.rs:238
             at /rustc/14997d56a550f4aa99fe737593cd2758227afc56/src/liballoc/boxed.rs:673
  14: __rust_maybe_catch_panic
             at src/libpanic_unwind/lib.rs:102
```

(This line:

https://github.com/tikv/tikv/blob/6aee17ab420f513b6c6c60329a00720766443d8d/tests/integrations/raftstore/test_conf_change.rs#L456

)

* Original log file: [log.txt](https://github.com/tikv/tikv/files/2953379/log.txt)

The PR is based on 6aee17ab. The PR only touches `src/import/engine.rs`, which cannot be the cause of the failure above.

**Comment by overvenus** [comment]
Another failed case 
[internal.pingcap.net.txt](https://github.com/tikv/tikv/files/3594423/internal.pingcap.net.txt)

**Comment by howaboutsynergy** [comment]
It fails intermittently, even when ran alone, like:
`$ set +e; counter=0; trap 'echo "Been $counter tries"' ERR; true; while test "$?" == "0"; do let 'counter++'; time RUST_BACKTRACE=1 RUSTFLAGS=-Cdebuginfo=2 cargo test --all raftstore::test_conf_change::test_server_split_brain -- --nocapture ; done `

<details>

```
!! master-installed (default)
!! Executing '/home/user/build/2nonpkgs/rust.stuff/cargo/cargo//target/release//cargo' in pwd='/home/user/build/2nonpkgs/rust.stuff/tikv' with args(5): 'test --all raftstore::test_conf_change::test_server_split_brain -- --nocapture'
    Finished dev [unoptimized + debuginfo] target(s) in 0.54s
     Running target/debug/deps/backup-248c27fd955a2fe7

running 0 tests

test result: ok. 0 passed; 0 failed; 0 ignored; 0 measured; 7 filtered out

     Running target/debug/deps/cmd-b9006753276863bc

running 0 tests

test result: ok. 0 passed; 0 failed; 0 ignored; 0 measured; 0 filtered out

     Running target/debug/deps/tikv_ctl-97d094d1325df856

running 0 tests

test result: ok. 0 passed; 0 failed; 0 ignored; 0 measured; 2 filtered out

     Running target/debug/deps/tikv_server-c23e19a6a06b8d3c

running 0 tests

test result: ok. 0 passed; 0 failed; 0 ignored; 0 measured; 0 filtered out

     Running target/debug/deps/codec-044d8c0f849c42ff

running 0 tests

test result: ok. 0 passed; 0 failed; 0 ignored; 0 measured; 82 filtered out

     Running target/debug/deps/engine-b188d984ef37defa

running 0 tests

test result: ok. 0 passed; 0 failed; 0 ignored; 0 measured; 25 filtered out

     Running target/debug/deps/external_storage-4988e07a9142350b

running 0 tests

test result: ok. 0 passed; 0 failed; 0 ignored; 0 measured; 3 filtered out

     Running target/debug/deps/fuzz-f3dff793f29c1988

running 0 tests

test result: ok. 0 passed; 0 failed; 0 ignored; 0 measured; 0 filtered out

     Running target/debug/deps/fuzz_targets-e1b24d370909a745

running 0 tests

test result: ok. 0 passed; 0 failed; 0 ignored; 0 measured; 0 filtered out

     Running target/debug/deps/fuzzer_afl-86ef046e621fda5c

running 0 tests

test result: ok. 0 passed; 0 failed; 0 ignored; 0 measured; 0 filtered out

     Running target/debug/deps/fuzzer_honggfuzz-5e5cd434dd44c0b1

running 0 tests

test result: ok. 0 passed; 0 failed; 0 ignored; 0 measured; 0 filtered out

     Running target/debug/deps/fuzzer_libfuzzer-0a320d96cf63c736

running 0 tests

test result: ok. 0 passed; 0 failed; 0 ignored; 0 measured; 0 filtered out

     Running target/debug/deps/keys-827caf0ee685ffee

running 0 tests

test result: ok. 0 passed; 0 failed; 0 ignored; 0 measured; 7 filtered out

     Running target/debug/deps/log_wrappers-27fd1652dd182864

running 0 tests

test result: ok. 0 passed; 0 failed; 0 ignored; 0 measured; 2 filtered out

     Running target/debug/deps/match_template-973ec91222b82bee

running 0 tests

test result: ok. 0 passed; 0 failed; 0 ignored; 0 measured; 2 filtered out

     Running target/debug/deps/panic_hook-445b56961b21beaa

running 0 tests

test result: ok. 0 passed; 0 failed; 0 ignored; 0 measured; 0 filtered out

     Running target/debug/deps/pd_client-e2e754210a98801c

running 0 tests

test result: ok. 0 passed; 0 failed; 0 ignored; 0 measured; 1 filtered out

     Running target/debug/deps/profiler-aa4b6882167b555d

running 0 tests

test result: ok. 0 passed; 0 failed; 0 ignored; 0 measured; 0 filtered out

     Running target/debug/deps/sst_importer-d66807a93b25ec5f

running 0 tests

test result: ok. 0 passed; 0 failed; 0 ignored; 0 measured; 4 filtered out

     Running target/debug/deps/test_coprocessor-1e07b971b322606f

running 0 tests

test result: ok. 0 passed; 0 failed; 0 ignored; 0 measured; 1 filtered out

     Running target/debug/deps/test_raftstore-0f0c58deb91e4637

running 0 tests

test result: ok. 0 passed; 0 failed; 0 ignored; 0 measured; 0 filtered out

     Running target/debug/deps/test_storage-26cf9f81a39c9600

running 0 tests

test result: ok. 0 passed; 0 failed; 0 ignored; 0 measured; 0 filtered out

     Running target/debug/deps/test_util-2820b084c43f6ca3

running 0 tests

test result: ok. 0 passed; 0 failed; 0 ignored; 0 measured; 1 filtered out

     Running target/debug/deps/tidb_query-da60cd5a9b0811bf

running 0 tests

test result: ok. 0 passed; 0 failed; 0 ignored; 0 measured; 508 filtered out

     Running target/debug/deps/tidb_query_codegen-635361f42feeabab

running 0 tests

test result: ok. 0 passed; 0 failed; 0 ignored; 0 measured; 11 filtered out

     Running target/debug/deps/tidb_query_datatype-19262fb96d973530

running 0 tests

test result: ok. 0 passed; 0 failed; 0 ignored; 0 measured; 0 filtered out

     Running target/debug/deps/tikv-9301ed19cd0d887d

running 0 tests

test result: ok. 0 passed; 0 failed; 0 ignored; 0 measured; 285 filtered out

     Running target/debug/deps/integrations-af4454caf70e82d5

running 1 test
glibc64:../sysdeps/posix/getaddrinfo.c:2201/getaddrinfo: integrations-af4454caf70e82d5[233277](full:'/home/user/build/2nonpkgs/rust.stuff/tikv/target/debug/deps/integrations-af4454caf70e82d5') for user user(1000(eff:user(1000))) 1of2 attempting to resolve (requested)hostname:
127.0.0.1
glibc64:../sysdeps/posix/getaddrinfo.c:2201/getaddrinfo: integrations-af4454caf70e82d5[233277](full:'/home/user/build/2nonpkgs/rust.stuff/tikv/target/debug/deps/integrations-af4454caf70e82d5') for user user(1000(eff:user(1000))) 2of2 successfully resolved requested hostname('127.0.0.1') which was not transformed('127.0.0.1') as follows:
127.0.0.1 127.0.0.1
glibc64:../sysdeps/posix/getaddrinfo.c:2201/getaddrinfo: integrations-af4454caf70e82d5[233277](full:'/home/user/build/2nonpkgs/rust.stuff/tikv/target/debug/deps/integrations-af4454caf70e82d5') for user user(1000(eff:user(1000))) 1of2 attempting to resolve (requested)hostname:
127.0.0.1
glibc64:../sysdeps/posix/getaddrinfo.c:2201/getaddrinfo: integrations-af4454caf70e82d5[233277](full:'/home/user/build/2nonpkgs/rust.stuff/tikv/target/debug/deps/integrations-af4454caf70e82d5') for user user(1000(eff:user(1000))) 2of2 successfully resolved requested hostname('127.0.0.1') which was not transformed('127.0.0.1') as follows:
127.0.0.1 127.0.0.1
glibc64:../sysdeps/posix/getaddrinfo.c:2201/getaddrinfo: integrations-af4454caf70e82d5[233277](full:'/home/user/build/2nonpkgs/rust.stuff/tikv/target/debug/deps/integrations-af4454caf70e82d5') for user user(1000(eff:user(1000))) 1of2 attempting to resolve (requested)hostname:
127.0.0.1
glibc64:../sysdeps/posix/getaddrinfo.c:2201/getaddrinfo: integrations-af4454caf70e82d5[233277](full:'/home/user/build/2nonpkgs/rust.stuff/tikv/target/debug/deps/integrations-af4454caf70e82d5') for user user(1000(eff:user(1000))) 2of2 successfully resolved requested hostname('127.0.0.1') which was not transformed('127.0.0.1') as follows:
127.0.0.1 127.0.0.1
glibc64:../sysdeps/posix/getaddrinfo.c:2201/getaddrinfo: integrations-af4454caf70e82d5[233277](full:'/home/user/build/2nonpkgs/rust.stuff/tikv/target/debug/deps/integrations-af4454caf70e82d5') for user user(1000(eff:user(1000))) 1of2 attempting to resolve (requested)hostname:
127.0.0.1
glibc64:../sysdeps/posix/getaddrinfo.c:2201/getaddrinfo: integrations-af4454caf70e82d5[233277](full:'/home/user/build/2nonpkgs/rust.stuff/tikv/target/debug/deps/integrations-af4454caf70e82d5') for user user(1000(eff:user(1000))) 2of2 successfully resolved requested hostname('127.0.0.1') which was not transformed('127.0.0.1') as follows:
127.0.0.1 127.0.0.1
glibc64:../sysdeps/posix/getaddrinfo.c:2201/getaddrinfo: integrations-af4454caf70e82d5[233277](full:'/home/user/build/2nonpkgs/rust.stuff/tikv/target/debug/deps/integrations-af4454caf70e82d5') for user user(1000(eff:user(1000))) 1of2 attempting to resolve (requested)hostname:
127.0.0.1
glibc64:../sysdeps/posix/getaddrinfo.c:2201/getaddrinfo: integrations-af4454caf70e82d5[233277](full:'/home/user/build/2nonpkgs/rust.stuff/tikv/target/debug/deps/integrations-af4454caf70e82d5') for user user(1000(eff:user(1000))) 2of2 successfully resolved requested hostname('127.0.0.1') which was not transformed('127.0.0.1') as follows:
127.0.0.1 127.0.0.1
glibc64:../sysdeps/posix/getaddrinfo.c:2201/getaddrinfo: integrations-af4454caf70e82d5[233277](full:'/home/user/build/2nonpkgs/rust.stuff/tikv/target/debug/deps/integrations-af4454caf70e82d5') for user user(1000(eff:user(1000))) 1of2 attempting to resolve (requested)hostname:
127.0.0.1
glibc64:../sysdeps/posix/getaddrinfo.c:2201/getaddrinfo: integrations-af4454caf70e82d5[233277](full:'/home/user/build/2nonpkgs/rust.stuff/tikv/target/debug/deps/integrations-af4454caf70e82d5') for user user(1000(eff:user(1000))) 2of2 successfully resolved requested hostname('127.0.0.1') which was not transformed('127.0.0.1') as follows:
127.0.0.1 127.0.0.1
glibc64:../sysdeps/posix/getaddrinfo.c:2201/getaddrinfo: integrations-af4454caf70e82d5[233277](full:'/home/user/build/2nonpkgs/rust.stuff/tikv/target/debug/deps/integrations-af4454caf70e82d5') for user user(1000(eff:user(1000))) 1of2 attempting to resolve (requested)hostname:
127.0.0.1
glibc64:../sysdeps/posix/getaddrinfo.c:2201/getaddrinfo: integrations-af4454caf70e82d5[233277](full:'/home/user/build/2nonpkgs/rust.stuff/tikv/target/debug/deps/integrations-af4454caf70e82d5') for user user(1000(eff:user(1000))) 2of2 successfully resolved requested hostname('127.0.0.1') which was not transformed('127.0.0.1') as follows:
127.0.0.1 127.0.0.1
glibc64:../sysdeps/posix/getaddrinfo.c:2201/getaddrinfo: integrations-af4454caf70e82d5[233277](full:'/home/user/build/2nonpkgs/rust.stuff/tikv/target/debug/deps/integrations-af4454caf70e82d5') for user user(1000(eff:user(1000))) 1of2 attempting to resolve (requested)hostname:
127.0.0.1
glibc64:../sysdeps/posix/getaddrinfo.c:2201/getaddrinfo: integrations-af4454caf70e82d5[233277](full:'/home/user/build/2nonpkgs/rust.stuff/tikv/target/debug/deps/integrations-af4454caf70e82d5') for user user(1000(eff:user(1000))) 2of2 successfully resolved requested hostname('127.0.0.1') which was not transformed('127.0.0.1') as follows:
127.0.0.1 127.0.0.1
glibc64:../sysdeps/posix/getaddrinfo.c:2201/getaddrinfo: integrations-af4454caf70e82d5[233277](full:'/home/user/build/2nonpkgs/rust.stuff/tikv/target/debug/deps/integrations-af4454caf70e82d5') for user user(1000(eff:user(1000))) 1of2 attempting to resolve (requested)hostname:
127.0.0.1
glibc64:../sysdeps/posix/getaddrinfo.c:2201/getaddrinfo: integrations-af4454caf70e82d5[233277](full:'/home/user/build/2nonpkgs/rust.stuff/tikv/target/debug/deps/integrations-af4454caf70e82d5') for user user(1000(eff:user(1000))) 2of2 successfully resolved requested hostname('127.0.0.1') which was not transformed('127.0.0.1') as follows:
127.0.0.1 127.0.0.1
glibc64:../sysdeps/posix/getaddrinfo.c:2201/getaddrinfo: integrations-af4454caf70e82d5[233277](full:'/home/user/build/2nonpkgs/rust.stuff/tikv/target/debug/deps/integrations-af4454caf70e82d5') for user user(1000(eff:user(1000))) 1of2 attempting to resolve (requested)hostname:
127.0.0.1
glibc64:../sysdeps/posix/getaddrinfo.c:2201/getaddrinfo: integrations-af4454caf70e82d5[233277](full:'/home/user/build/2nonpkgs/rust.stuff/tikv/target/debug/deps/integrations-af4454caf70e82d5') for user user(1000(eff:user(1000))) 2of2 successfully resolved requested hostname('127.0.0.1') which was not transformed('127.0.0.1') as follows:
127.0.0.1 127.0.0.1
glibc64:../sysdeps/posix/getaddrinfo.c:2201/getaddrinfo: integrations-af4454caf70e82d5[233277](full:'/home/user/build/2nonpkgs/rust.stuff/tikv/target/debug/deps/integrations-af4454caf70e82d5') for user user(1000(eff:user(1000))) 1of2 attempting to resolve (requested)hostname:
127.0.0.1
glibc64:../sysdeps/posix/getaddrinfo.c:2201/getaddrinfo: integrations-af4454caf70e82d5[233277](full:'/home/user/build/2nonpkgs/rust.stuff/tikv/target/debug/deps/integrations-af4454caf70e82d5') for user user(1000(eff:user(1000))) 2of2 successfully resolved requested hostname('127.0.0.1') which was not transformed('127.0.0.1') as follows:
127.0.0.1 127.0.0.1
glibc64:../sysdeps/posix/getaddrinfo.c:2201/getaddrinfo: integrations-af4454caf70e82d5[233277](full:'/home/user/build/2nonpkgs/rust.stuff/tikv/target/debug/deps/integrations-af4454caf70e82d5') for user user(1000(eff:user(1000))) 1of2 attempting to resolve (requested)hostname:
127.0.0.1
glibc64:../sysdeps/posix/getaddrinfo.c:2201/getaddrinfo: integrations-af4454caf70e82d5[233277](full:'/home/user/build/2nonpkgs/rust.stuff/tikv/target/debug/deps/integrations-af4454caf70e82d5') for user user(1000(eff:user(1000))) 2of2 successfully resolved requested hostname('127.0.0.1') which was not transformed('127.0.0.1') as follows:
127.0.0.1 127.0.0.1
glibc64:../sysdeps/posix/getaddrinfo.c:2201/getaddrinfo: integrations-af4454caf70e82d5[233277](full:'/home/user/build/2nonpkgs/rust.stuff/tikv/target/debug/deps/integrations-af4454caf70e82d5') for user user(1000(eff:user(1000))) 1of2 attempting to resolve (requested)hostname:
127.0.0.1
glibc64:../sysdeps/posix/getaddrinfo.c:2201/getaddrinfo: integrations-af4454caf70e82d5[233277](full:'/home/user/build/2nonpkgs/rust.stuff/tikv/target/debug/deps/integrations-af4454caf70e82d5') for user user(1000(eff:user(1000))) 2of2 successfully resolved requested hostname('127.0.0.1') which was not transformed('127.0.0.1') as follows:
127.0.0.1 127.0.0.1
glibc64:../sysdeps/posix/getaddrinfo.c:2201/getaddrinfo: integrations-af4454caf70e82d5[233277](full:'/home/user/build/2nonpkgs/rust.stuff/tikv/target/debug/deps/integrations-af4454caf70e82d5') for user user(1000(eff:user(1000))) 1of2 attempting to resolve (requested)hostname:
127.0.0.1
glibc64:../sysdeps/posix/getaddrinfo.c:2201/getaddrinfo: integrations-af4454caf70e82d5[233277](full:'/home/user/build/2nonpkgs/rust.stuff/tikv/target/debug/deps/integrations-af4454caf70e82d5') for user user(1000(eff:user(1000))) 2of2 successfully resolved requested hostname('127.0.0.1') which was not transformed('127.0.0.1') as follows:
127.0.0.1 127.0.0.1
thread 'raftstore::test_conf_change::test_server_split_brain' panicked at 'region Some(id: 1 region_epoch { conf_ver: 3 version: 1 } peers { id: 1 store_id: 1 } peers { id: 2 store_id: 2 } peers { id: 3 store_id: 3 }) has no peer id: 4 store_id: 4', /home/user/build/2nonpkgs/rust.stuff/tikv/components/test_raftstore/src/pd.rs:753:9
stack backtrace:
   0: backtrace::backtrace::libunwind::trace
             at /home/user/.cargo/registry/src/github.com-1ecc6299db9ec823/backtrace-0.3.35/src/backtrace/libunwind.rs:88
   1: backtrace::backtrace::trace_unsynchronized
             at /home/user/.cargo/registry/src/github.com-1ecc6299db9ec823/backtrace-0.3.35/src/backtrace/mod.rs:66
   2: std::sys_common::backtrace::_print
             at src/libstd/sys_common/backtrace.rs:47
   3: std::sys_common::backtrace::print
             at src/libstd/sys_common/backtrace.rs:36
   4: std::panicking::default_hook::{{closure}}
             at src/libstd/panicking.rs:200
   5: std::panicking::default_hook
             at src/libstd/panicking.rs:214
   6: std::panicking::rust_panic_with_hook
             at src/libstd/panicking.rs:477
   7: std::panicking::continue_panic_fmt
             at src/libstd/panicking.rs:384
   8: std::panicking::begin_panic_fmt
             at src/libstd/panicking.rs:339
   9: test_raftstore::pd::TestPdClient::must_have_peer
             at ./components/test_raftstore/src/pd.rs:753
  10: test_raftstore::pd::TestPdClient::must_add_peer
             at ./components/test_raftstore/src/pd.rs:850
  11: integrations::raftstore::test_conf_change::test_split_brain
             at ./tests/integrations/raftstore/test_conf_change.rs:442
  12: integrations::raftstore::test_conf_change::test_server_split_brain
             at ./tests/integrations/raftstore/test_conf_change.rs:498
  13: integrations::raftstore::test_conf_change::test_server_split_brain::{{closure}}
             at ./tests/integrations/raftstore/test_conf_change.rs:495
  14: core::ops::function::FnOnce::call_once
             at /home/user/build/2nonpkgs/rust.stuff/rust/rust/src/libcore/ops/function.rs:227
  15: <alloc::boxed::Box<F> as core::ops::function::FnOnce<A>>::call_once
             at /home/user/build/2nonpkgs/rust.stuff/rust/rust/src/liballoc/boxed.rs:922
  16: __rust_maybe_catch_panic
             at src/libpanic_unwind/lib.rs:80
  17: std::panicking::try
             at /home/user/build/2nonpkgs/rust.stuff/rust/rust/src/libstd/panicking.rs:275
  18: test::run_test::run_test_inner::{{closure}}
             at src/libtest/lib.rs:1413
note: Some details are omitted, run with `RUST_BACKTRACE=full` for a verbose backtrace.
glibc64:../sysdeps/posix/getaddrinfo.c:2201/getaddrinfo: integrations-af4454caf70e82d5[233277](full:'/home/user/build/2nonpkgs/rust.stuff/tikv/target/debug/deps/integrations-af4454caf70e82d5') for user user(1000(eff:user(1000))) 1of2 attempting to resolve (requested)hostname:
127.0.0.1
glibc64:../sysdeps/posix/getaddrinfo.c:2201/getaddrinfo: integrations-af4454caf70e82d5[233277](full:'/home/user/build/2nonpkgs/rust.stuff/tikv/target/debug/deps/integrations-af4454caf70e82d5') for user user(1000(eff:user(1000))) 2of2 successfully resolved requested hostname('127.0.0.1') which was not transformed('127.0.0.1') as follows:
127.0.0.1 127.0.0.1
glibc64:../sysdeps/posix/getaddrinfo.c:2201/getaddrinfo: integrations-af4454caf70e82d5[233277](full:'/home/user/build/2nonpkgs/rust.stuff/tikv/target/debug/deps/integrations-af4454caf70e82d5') for user user(1000(eff:user(1000))) 1of2 attempting to resolve (requested)hostname:
127.0.0.1
glibc64:../sysdeps/posix/getaddrinfo.c:2201/getaddrinfo: integrations-af4454caf70e82d5[233277](full:'/home/user/build/2nonpkgs/rust.stuff/tikv/target/debug/deps/integrations-af4454caf70e82d5') for user user(1000(eff:user(1000))) 2of2 successfully resolved requested hostname('127.0.0.1') which was not transformed('127.0.0.1') as follows:
127.0.0.1 127.0.0.1
glibc64:../sysdeps/posix/getaddrinfo.c:2201/getaddrinfo: integrations-af4454caf70e82d5[233277](full:'/home/user/build/2nonpkgs/rust.stuff/tikv/target/debug/deps/integrations-af4454caf70e82d5') for user user(1000(eff:user(1000))) 1of2 attempting to resolve (requested)hostname:
127.0.0.1
glibc64:../sysdeps/posix/getaddrinfo.c:2201/getaddrinfo: integrations-af4454caf70e82d5[233277](full:'/home/user/build/2nonpkgs/rust.stuff/tikv/target/debug/deps/integrations-af4454caf70e82d5') for user user(1000(eff:user(1000))) 2of2 successfully resolved requested hostname('127.0.0.1') which was not transformed('127.0.0.1') as follows:
127.0.0.1 127.0.0.1
test raftstore::test_conf_change::test_server_split_brain ... FAILED

failures:

failures:
    raftstore::test_conf_change::test_server_split_brain

test result: FAILED. 0 passed; 1 failed; 0 ignored; 0 measured; 288 filtered out

error: test failed, to rerun pass '-p tikv --test integrations'
Been 25 tries

real	0m9.082s
user	0m8.543s
sys	0m2.321s
```

</details>

**Comment by hicqu** [comment]
Stablized by #5719.
